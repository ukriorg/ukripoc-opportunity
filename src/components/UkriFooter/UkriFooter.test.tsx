import React from "react";
import renderer from "react-test-renderer";
import { UkriFooter } from "./index";

describe("UkriFooter", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<UkriFooter />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});

