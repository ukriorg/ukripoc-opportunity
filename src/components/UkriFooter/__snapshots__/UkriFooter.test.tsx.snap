// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UkriFooter renders correctly 1`] = `
<div
  className="sc-bwzfXH iTlVXu"
>
  <div
    className="sc-htpNat jtpwyQ"
  >
    <div>
      <a
        className="sc-bdVaJa huzbaj"
        href="/help"
        muted={false}
      >
        Help
      </a>
      <a
        className="sc-bdVaJa huzbaj"
        href="/help"
        muted={false}
      >
        UKRI public website
      </a>
      <a
        className="sc-bdVaJa huzbaj"
        href="/terms"
        muted={false}
      >
        Terms and conditions
      </a>
    </div>
  </div>
</div>
`;
